Makefile

```diff
@@ -18,7 +22,8 @@ remote-build:
 	@cd controller && docker run --rm -w /go/src/github.com/shipyard/shipyard --entrypoint /bin/bash shipyard-build -c "make build 1>&2 && cd controller && tar -czf - controller" | tar zxf -
 
 media:
-	@cd controller/static && npm install && npm run build
+	docker build -t $(UI_BUILD_IMAGE) -f ui/Dockerfile.build ui && \
```

Comment:
Media builds are done in a container now and produce a tarball for the built UI bits.